apps/api/src/services/data-provider/coingecko/coingecko.service.ts

```diff
@@ -17,15 +17,25 @@ import {
   SymbolProfile
 } from '@prisma/client';
 import { format, fromUnixTime, getUnixTime } from 'date-fns';
-import got from 'got';
+import got, { Headers } from 'got';
 
 @Injectable()
 export class CoinGeckoService implements DataProviderInterface {
-  private readonly URL = 'https://api.coingecko.com/api/v3';
+  private headers: Headers = {};
+  private readonly URL: string = 'https://api.coingecko.com/api/v3';
```

Comment:
Can you refactor to `private url = 'https://api.coingecko.com/api/v3';` and move up to keep the alphabetical order?

It is not `readonly` anymore and you don't need to set the type due to the initial value.

Reply:
I left it to `readonly` since TypeScript allows [modifications under the constructor](https://www.tutorialsteacher.com/typescript/typescript-readonly) for such variable.

The added type declaration is to prevent the linter/transpiler from complaining.

Let me know if you would rather have it as `private url` either way.

Also, isn't it already alphabetically ordered that way ?